generated/nifake/custom_struct.py

```diff
@@ -0,0 +1,37 @@
+import ctypes
+
+from nifake import visatype
+
+
+# This class is an internal implementation detail
+class _CustomStructCType(ctypes.Structure):
+    _fields_ = [
+        ('struct_int', visatype.ViInt32),
+        ('struct_double', visatype.ViReal64),
+    ]
+
+    def __init__(self, data=None, struct_int=0, struct_double=0.0):
+        super(ctypes.Structure, self).__init__()
+        if data is not None:
+            self.struct_int = data.struct_int
+            self.struct_double = data.struct_double
+        else:
+            self.struct_int = struct_int
+            self.struct_double = struct_double
+
+
+# We need to match the name in the function definition
```

Comment:
I don't understand why this is necessary. Can you explain just for my benefit? You mentioned something about making flake8 happy.

Reply:
The part about making flake8 happy is we need `custom_struct` to be the (or at least a) name for the type. This name doesn't follow the convention of CamelCase so flake8 would complain if that was the name of the class. To make flake8 happy, I gave the class a proper name and then assigned it to the name we needed in the API.

I removed `custom_struct = _CustomStructCType` and just called said `class custom_struct`. This did require a noqa statement to tell flake8 not to complain about it.